pos_quick_logout/models/pos_quick_logout.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
```

Comment:
please rename the file into pos_config.py

Reply:
done